packages/caip/src/adapters/banxa/index.ts

```diff
@@ -20,6 +20,19 @@ import {
   polygonChainId,
 } from '../../constants'
 
+// These assets are in theory supported by Banxa, however only matic works as expected
+const banxaPolygonAssets = {
+  [polygonAssetId]: 'matic',
+  // 'eip155:137/erc20:0x3cef98bb43d732e2f285ee605a8158cde967d219': 'bat',
+  // 'eip155:137/erc20:0x8505b9d2254a7ae468c0e9dd10ccea3a837aef5c': 'comp',
+  // 'eip155:137/erc20:0x8f3cf7ad23cd3cadbd9735aff958023239c6a063': 'dai',
+  // 'eip155:137/erc20:0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39': 'link',
+  // 'eip155:137/erc20:0x50b728d8d964fd00c2d0aad81718b71311fef68a': 'snx',
+  // 'eip155:137/erc20:0x2791bca1f2de4661ed88a30c99a7a9449aa84174': 'usdc',
+  // 'eip155:137/erc20:0xc2132d05d31c914a87c6611c10748aeb04b58e8f': 'usdt',
+  // 'eip155:137/erc20:0xda537104d6a5edd53c6fbba9a898708e465260b6': 'yfi',
+}
```

Comment:
This isn't really part of this diff, can be authored with the Polygon work. Usually trying not to be nitpicky in granular diffs as it's always a good thing to bring improvements whilst in the house, but given it was already a bit tricky to understand the intent of this PR, this made it look like this was also solving something (which it isn't).

Can we revert it?

Reply:
And let's remove `closes https://github.com/shapeshift/web/issues/4314` from the description after this is reverted

Reply:
agree w/ both of the above